src/lwc/rd2PauseForm/rd2PauseForm.html

```diff
@@ -67,14 +68,25 @@
 
                         <lightning-layout-item padding="horizontal-large" class="slds-p-top_large" size="12">
                             <template if:true={selectedRowsSummary}>
-                                <lightning-formatted-rich-text value={selectedRowsSummary} data-id="rowSummary"
-                                    data-qa-locator="richtext Row Summary">
-                                </lightning-formatted-rich-text>
+                                <div style="font-style: italic;">
+                                    <lightning-formatted-rich-text value={selectedRowsSummary} data-id="rowSummary"
```

Comment:
Just a suggestion. Since `<lightning-formatted-rich-text>` is used, instead of having a `<div>` with in line style attribute. Maybe you can append `<em></em> `tag to `{selectedRowsSummary}`. Same for line 81 to 86. What do you think?